src/styles/utils/hover.styl

```
@@ -0,0 +1,6 @@
+/**
+ * Media query to deal with hover on mobile devices - it disables the hover behaviour on mobile devices
+ */
+hover()
```

To think again about the naming, we can use a longer but clearer name, such as:
```suggestion
hover-desktop-only()
```
what do you think ?